src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -2390,4 +2390,13 @@ Uninstall the following packages: {0}?</value>
   <data name="PreferencesViewEnableNodeAutoCompleteTooltipText" xml:space="preserve">
     <value>Learn more about Node Autocomplete feature.</value>
   </data>
+  <data name="AddStyleButton" xml:space="preserve">
+    <value>Add Style</value>
+  </data>
+  <data name="ResetCPythonButtonText" xml:space="preserve">
+    <value>Reset CPython</value>
+  </data>
+  <data name="ResetCPythonButtonToolTip" xml:space="preserve">
+    <value>Resets CPython environment by reloading modules.</value>
```

Comment:
does this need to be more descriptive ?
all modules ? or some ? do we need to be specific I wonder ...

Reply:
thanks @pinzart90 - I was thinking to leave it a bit intentionally vague at this point so we can adjust it easily, but we could specify modules that have file attributes?

Reply:
not sure....maybe vague here and more specific in some dynamoDS wiki ?

Reply:
I've added https://github.com/DynamoDS/Dynamo/wiki/Work-in-progress-to-improve-Python-3-support for docs, we can expand on specifics there and easily update it.